src/comment.js

```diff
@@ -0,0 +1,6 @@
+function createCommentBody(langToolResult) {
+    const commentBody = `Total mistakes: ${langToolResult.matches.length}`;
```

Comment:
The complete implementation for the comment body was going to be done in the next branch. I'll show here what LanguageTool returns then show what I had in mind:

```
"matches": [
        {
            "message": "This word is normally spelled with a hyphen.",
            "shortMessage": "Compound",
            "replacements": [
                {
                    "value": "long-established"
                }
            ],
            "offset": 72,
            "length": 16,
            "context": {
                "text": "... This is the first paragraph. ● It is a long established fact that a reader. ● It is a long esta...",
                "offset": 43,
                "length": 16
            },
            "sentence": "● It is a long established fact that a reader.",
            "type": {
                "typeName": "Other"
            },
            "rule": {
                "id": "EN_COMPOUNDS_LONG_ESTABLISHED",
                "description": "Hyphenated words: long established",
                "issueType": "misspelling",
                "category": {
                    "id": "MISC",
                    "name": "Miscellaneous"
                }
            },
            "ignoreForIncompleteSentence": false,
            "contextForSureMatch": 0
        }
]
```

For no mistakes, I will implement your suggestion. For mistakes, I was thinking to first show number of mistakes, then for every mistake show `message` then `context.text` then [replacements](url) limited to 3 because for some spelling mistakes I have seen about 40 suggestions being returned.

Reply:
That's sensible